### DISCUSSION TITLE
Can we add the choice field in model?

### DISCUSSION THREAD

**Discussion by harshalizode** [dymmond/mongoz#37] [Q&A]
Hello,
This is a basic need for including the choice field in the model. Is there a way to add this, like Django?
I have choice field as: 
```
status = {
    ("A", "Active"),
    ("I", "Inactive")
}
```
Thank you

**Comment by tarsil** [comment]
@harshalizode of course. We just need to understand what is the corresponding object type in mongo and add it. We can release that with the other fix next week

**Comment by tarsil** [comment]
@harshalizode this repo will be moved to Dymmond so in a few minutes you will access via https://github.com/dymmond/mongoz

Just for FYI 👍🏼

**Comment by tarsil** [comment]
@harshalizode as you can see, now Mongoz is under Dymmond and the URL is https://mongoz.dymmond.com

**Comment by harshalizode** [comment]
@tarsil, is the choices field implemented?

**Reply by harshalizode** [reply]
I examined the newest version of esmerald, and the FieldFactory now contains the choices argument, however there is no example of how to use it or which data type we will use to implement the field as a choice field.

**Comment by harshalizode** [comment]
@tarsil, Is any update for the choice field in the Mongoz?

**Comment by harshalizode** [comment]
@tarsil, Is any update for the choice field in the Mongoz?
As I seen the multiple features we have provided in the FieldFactory, but not utilize into the provided fields to set those properties as defined into the FieldFactory.
The properties that we are providing into BaseField as:
* Choices
* read_only
* Sparse
* help_text
* alias
* description
* title

**Comment by tarsil** [comment]
@harshalizode it's in the pipeline but we need time to get there because of the whole ecosystem. We have a lot of tools to maintain and we don't get anything from it in the sense of allowing us 100% focus. We can only do with our spare time but soon this should be out. I just need to see how to get there

**Reply by harshalizode** [reply]
Okay, I understand. I appreciate your response and time.
